src/xdocs/config_whitespace.xml

```diff
@@ -1687,6 +1687,93 @@ Lists.charactersOf("foo")
         </p>
       </subsection>
     </section>
+    <section name="NoWhitespaceBeforeCaseDefaultColon">
+      <p>Since Checkstyle 8.43</p>
```

Comment:
please bump version here and in javadoc.

Reply:
Done